### PR TITLE
feat: remove 4.x support & fix test

### PR DESCRIPTION
**Pull request by atian25**
##### Checklist

- [ ] `npm test` passes
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message follows commit guidelines

### CODE REVIEW

File: .autod.conf.js [hunk 1]
```diff
@@ -4,8 +4,8 @@ module.exports = {
   write: true,
   prefix: '^',
   test: [
-    'test',
-    'benchmark',
+   'test',
```
Comment (fengmk2) [inline .autod.conf.js]: 缩进少了一个空格

File: README.md [hunk 2]
```diff
@@ -22,6 +22,8 @@
 
 Wrap [koa-etag](https://github.com/koajs/etag) and [koa-conditional-get](https://github.com/koajs/conditional-get) for egg
 
+**We recommend to use nginx to do this for better performance rather than use this plugin.**
```
Comment (fengmk2) [inline README.md]: `[nginx etag](http://nginx.org/en/docs/http/ngx_http_core_module.html#etag)`

File: package.json [hunk 3]
```diff
@@ -7,22 +7,21 @@
   },
   "keywords": [
     "egg",
-    "eggPlugin",
-    "egg-plugin"
```
Comment (fengmk2) [inline package.json]: 为啥删除这些关键词？不是方便 npm 搜索吗？

File: README.zh_CN.md [hunk 4]
```diff
@@ -26,6 +26,8 @@
 
 此功能在一些内容变化不多的场景，能有效避免网络下载。
 
+**通常来说，基于性能方面的考虑，在生产环境我们会使用 nginx 替代该插件来计算 etag 。**
```
Comment (huacnlee) [inline README.zh_CN.md]: nginx -> Nginx 
etag -> ETag

**Comment by atian25** [comment]
done, review again

**Comment by atian25** [comment]
ping @huacnlee, need more +1

File: README.md [hunk 5]
```diff
@@ -22,6 +22,8 @@
 
 Wrap [koa-etag](https://github.com/koajs/etag) and [koa-conditional-get](https://github.com/koajs/conditional-get) for egg
 
+**For better performance, we recommend to use [Nginx Etag](http://nginx.org/en/docs/http/ngx_http_core_module.html#etag) rather than use this plugin.**
```
Comment (huacnlee) [inline README.md]: Etag -> ETag

Comment (atian25) [inline README.md]: done

**Pull request merged**